Example/Tests/Classes/LeanplumTest.m

```diff
@@ -1122,6 +1166,70 @@ - (void)test_variables
     XCTAssertEqualObjects(@100LL, [long_long_variable defaultValue]);
 }
 
+/**
+ * Tests variant debug info.
+ */
+- (void)testStartResponseShouldParseVariantDebugInfo
+{
+    [Leanplum setVariantDebugInfoEnabled:YES];
+    // This stub have to be removed when start command is successfully executed.
+    id<OHHTTPStubsDescriptor> startStub = [OHHTTPStubs stubRequestsPassingTest:
+                                           ^BOOL(NSURLRequest * _Nonnull request) {
+                                               return [request.URL.host isEqualToString:API_HOST];
+                                           } withStubResponse:^OHHTTPStubsResponse * _Nonnull(NSURLRequest * _Nonnull request) {
+                                               NSString* response_file = OHPathForFile(@"start_with_variant_debug_info_response.json", self.class);
+                                               return [OHHTTPStubsResponse responseWithFileAtPath:response_file statusCode:200
+                                                                                          headers:@{@"Content-Type":@"application/json"}];
+                                           }];
+    
+    [LeanplumHelper setup_development_test];
+    dispatch_semaphore_t semaphore = dispatch_semaphore_create(0);
+    [Leanplum startWithResponseHandler:^(BOOL success) {
+        XCTAssertTrue(success);
+        [OHHTTPStubs removeStub:startStub];
+        XCTAssertNotNil([LPVarCache variantDebugInfo]);
+        NSDictionary *abTests = [LPVarCache variantDebugInfo][@"abTests"];
+        XCTAssertEqual(abTests.count, 2);
+        [LPVarCache saveDiffs];
+        [LPVarCache setVariantDebugInfo:nil];
+        XCTAssertNil([LPVarCache variantDebugInfo]);
+        [LPVarCache loadDiffs];
+        XCTAssertNotNil([LPVarCache variantDebugInfo]);
+        dispatch_semaphore_signal(semaphore);
+    }];
+    dispatch_semaphore_wait(semaphore, [LeanplumHelper default_dispatch_time]);
+    
+    // Remove stub after start is successful, so we don't capture requests from other methods
+    [OHHTTPStubs stubRequestsPassingTest:^BOOL(NSURLRequest * _Nonnull request) {
+        return [request.URL.host isEqualToString:API_HOST];
+    } withStubResponse:^OHHTTPStubsResponse * _Nonnull(NSURLRequest * _Nonnull request) {
+        NSString* response_file = OHPathForFile(@"variables_response.json", self.class);
```

Comment:
should we always be returning variables_response.json in other tests ?
is any other test doing this stubbing ?